azext_iot/digitaltwins/providers/resource.py

```diff
@@ -141,9 +141,6 @@ def get(self, name, resource_group_name, wait=False):
                 resource_name=name, resource_group_name=resource_group_name
             )
         except ErrorResponseException as e:
-            if wait:
```

Comment:
Let's ensure wait operations are not impacted cc: @vilit1

Reply:
they are not - the wait commands will wait for the resource to have a terminating provisioning state as before.